code/model/ErrorPage.php

```diff
@@ -47,36 +69,32 @@ public function canAddChildren($member = null) {
 	 * file generated when the user hit's save and publish in the CMS
 	 *
 	 * @param int $statusCode
-	 *
 	 * @return SS_HTTPResponse
 	 */
 	public static function response_for($statusCode) {
 		// first attempt to dynamically generate the error page
-		$errorPage = ErrorPage::get()->filter(array(
-			"ErrorCode" => $statusCode
-		))->first(); 
+		$errorPage = ErrorPage::get()
+			->filter(array(
+				"ErrorCode" => $statusCode
+			))->first(); 
 
 		if($errorPage) {
 			Requirements::clear();
 			Requirements::clear_combined_files();
 
-			return ModelAsController::controller_for($errorPage)->handleRequest(
-				new SS_HTTPRequest('GET', ''), DataModel::inst()
-			);
+			return ModelAsController::controller_for($errorPage)
+				->handleRequest(
+					new SS_HTTPRequest('GET', ''),
+					DataModel::inst()
+				);
 		}
 		
 		// then fall back on a cached version
-		$cachedPath = self::get_filepath_for_errorcode(
-			$statusCode, 
-			class_exists('Translatable') ? Translatable::get_current_locale() : null
```

Comment:
Will `Translatable::updateErrorFilename()` completely replace the need for this coupling? If so, could we add a GitHub issue for it in translatable, as well as subsites?